tabpfn_client/client.py

```diff
@@ -193,6 +193,8 @@ def _validate_response(
         # Read response.
         load = None
         try:
+            if not response.is_closed:
```

Comment:
what does this do? could you add a comment why you do this?

Reply:
This is to make sure that the response is read before response.json() gets called. When dealing with a streaming response (like during data download), I sometimes got an httpx.ResponseNotRead error without this if block. I will add a comment.

Reply:
Oh interesting. Thank you!